### PR TITLE
Final round of fixes for v1.0.0-beta.7

### PR DESCRIPTION
**Pull request by paulcpederson**
Several more bugs fixed in prep for v1.0.0-beta.7 + some small doc fixes as well. 

This also rolls back to using fonts.com for the web fonts until the point where we can use monotype's solution reliably.

### CODE REVIEW

File: lib/js/helpers/classy.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function has (domNode, className) {
 export function add (domNode, classes) {
   classes.split(' ').forEach(function (c) {
     if (!has(domNode, c)) {
-      var existingClass = domNode.getAttribute('class') || "";
+      var existingClass = domNode.getAttribute('class') || '';
```
Comment (jonathantneal) [inline lib/js/helpers/classy.js]: Your browser support is IE10 and up now, yes? http://caniuse.com/#search=classList

File: lib/js/patterns/modal.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ import bus from '../helpers/bus';
 
 function modal () {
   // Cool nodes
+  var html = document.querySelector('html');
```
Comment (jonathantneal) [inline lib/js/patterns/modal.js]: `document.documentElement`?

File: docs/source/assets/css/all.scss [hunk 3]
```diff
@@ -19,24 +19,11 @@ $include-calcite-icons: false;
...
-  background-image: linear-gradient(110deg, #005e95 10%, #9bc19c 90%);
-  -webkit-text-fill-color: transparent;
-  -webkit-background-clip: text;
-  -moz-background-clip: text;
-  background-clip: text;
-}
-
 .huge-text {
-  font-size: 5vw;
+  font-size: calc(4vw + 16px);
```
Comment (jonathantneal) [inline docs/source/assets/css/all.scss]: This commit seems to be doing more than updating homepage banner text. Is `.huge-text` a global?

File: lib/sass/calcite-web/type/_config.scss [hunk 4]
```diff
@@ -30,7 +30,7 @@ $indent:             1em;
 
 // Header Family
 $avenir-tracking:    0;
-$avenir-family:      'Avenir Next', 'Helvetica Neue', sans-serif;
+$avenir-family:      'Avenir Next W01', 'Avenir Next W00', 'Avenir Next', 'Avenir', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
```
Comment (jonathantneal) [inline lib/sass/calcite-web/type/_config.scss]: Did you mean to also restore the (unnecessary, AFAIK) `'Helvetica', 'Arial'`?

**Comment by paulcpederson** [comment]
@jonathantneal @oknoway if either of you want to take a gander again I would appreciate it

File: lib/sass/calcite-web/type/_config.scss (hunk 4 above)
Comment (paulcpederson) [inline lib/sass/calcite-web/type/_config.scss]: I just reverted, but good idea, I'll remove those

File: docs/source/assets/css/all.scss (hunk 3 above)
Comment (paulcpederson) [inline docs/source/assets/css/all.scss]: No, that Sass is not part of calcite-web at all, just for the docs.

File: lib/js/helpers/classy.js (hunk 1 above)
Comment (paulcpederson) [inline lib/js/helpers/classy.js]: Yeah, good idea. I'll open an issue and we can address that before moving to v1.0

File: lib/js/patterns/modal.js (hunk 2 above)
Comment (paulcpederson) [inline lib/js/patterns/modal.js]: Nice, that is much cleaner.

File: docs/source/assets/css/all.scss (hunk 3 above)
Comment (jonathantneal) [inline docs/source/assets/css/all.scss]: Consider this one resolved. 👍

**Comment by paulcpederson** [comment]
@jonathantneal ok, I think I got all of those. Thanks so much for reviewing 🌈

**Pull request merged**